templates/index.html

```diff
@@ -24,6 +24,12 @@ <h2 align="center">Meilix Generator</h2>
                         <label for="TRAVIS_TAG">Event Name:</label>
                         <input pattern="[A-Za-z0-9\s]{1,20}" class="form-control" placeholder="Enter event" name="TRAVIS_TAG" required="" type="text">
                     </div>
+                  
+                    <div class="form-group">
+                      <label for="event_url">Event Link:</label>
+                      <input pattern="https?://.+" class="form-control" placeholder="Enter event link" name="event_url" required="" type="url" >
```

Comment:
Most of the websites are not SSL certified. So better we can go for http:// only.